src/content/basics/ha-masters/index.md

```diff
@@ -13,56 +13,56 @@ categories: ["basics"]
 
 Kubernetes master nodes are the nodes that run the Kubernetes API of a tenant cluster,
 as well as some other important components. In the case of Giant Swarm, the master nodes
-also host the [etcd](https://etcd.io/) database that keeps all the state that configures
+also host the [etcd](https://etcd.io/) database that keeps all the states that configure
 the cluster, the workloads and other resources.
 
-Clusters can run in a fully functional way with one master node, however this renders the
+Clusters can run in a fully functional way with one master node. However this renders the
 Kubernetes API unavailable in certain cases like a cluster upgrade or even an outage of
 the underlying infrastructure.
 
 With AWS release v{{% first_aws_ha_masters_version %}} new tenant clusters are launched
-with three master nodes in different availability zones by default, increasing the API
-availability during upgrades and cluster changes drastically and making the cluster more
+with three master nodes in different availability zones by default. Thus, increasing the API
+availability during upgrades and drastic changes to the cluster. It also makes the cluster more
```

Comment:
This is not what I wanted to express. The intended meaning was: during changes to the cluster, the availability is increased to a great extend. Or, in other words, downtimes will be reduced a great deal.

Reply:
Do you want to try rewriting it in a way that aligns with the message you would like to deliver?

Reply:
Let's try again :)

> With AWS release v{{% first_aws_ha_masters_version %}}, new tenant clusters are launched with three master nodes by default, spread over multiple availability zones. This results in much higher availability of the API during upgrades or other changes to the cluster configuration, plus enhanced resilience against any data center or hardware failures.

Reply:
Looks good.

Reply:
Thanks!